src/main/java/org/jfrog/hudson/pipeline/scripted/steps/CreateGoBuildStep.java

```diff
@@ -13,9 +12,14 @@
     public CreateGoBuildStep() {
     }
 
-    public static class Execution extends AbstractSynchronousStepExecution<GoBuild> {
+    public static class Execution extends SynchronousStepExecution<GoBuild> {
```

Comment:
Don't you prefer to extend ArtifactorySynchronousStepExecution here also? (and in additional steps you're extending SynchronousStepExecution directly).

Reply:
We added a comment that explains why we doesn't extend ArtifactorySynchronousStepExecution.